test/Riok.Mapperly.Tests/TestSourceBuilder.cs

```diff
@@ -59,6 +67,32 @@ public static string MapperWithBody([StringSyntax(StringSyntax.CSharp)] string b
         );
     }
 
+    public static string MapperWithBodyInBlockScopedNamespace(
+        [StringSyntax(StringSyntax.CSharp)] string body,
+        TestSourceBuilderOptions? options = null
```

Comment:
I added some new things to TestSourceBuilder, so I can test the mapper behavior when classes are in different namespaces.